packages/edit-site/src/components/global-styles/preview.js

```diff
@@ -82,7 +82,17 @@ const StylesPreview = ( { label, isFocused } ) => {
 	return (
 		<Iframe
 			className="edit-site-global-styles-preview__iframe"
-			head={ <EditorStyles styles={ styles } /> }
+			head={
+				<EditorStyles
+					styles={ [
+						...styles,
+						{
+							css: 'body{min-width: 0;}',
```

Comment:
Should we do it in the iframe component instead?

Reply:
I guess if we can't do it without overriding theme styles, the current PR as it stands is good enough because it limits it to the preview. So maybe let's just add a comment.

Reply:
Also, there's a performance regression here, we will be retransforming styles on each render because of the new array instance on each render which can be impactful.

Reply:
Added the comment.

This issue currently only affects this component, so I think fixing it locally makes more sense.

Reply:
> Also, there's a performance regression here, we will be retransforming styles on each render because of the new array instance on each render which can be impactful.

I can memoize merged styles.

Reply:
Done ✅